### DISCUSSION TITLE
slide-level in YAML metadata

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [jgm/pandoc#9764] [Ideas]
I request the possibility to specify the [`--slide-level` option for slideshows](https://pandoc.org/chunkedhtml-demo/10.1-structuring-the-slide-show.html) with the [YAML metadata for beamer](https://pandoc.org/chunkedhtml-demo/6.2-variables.html#variables-for-beamer-slides).

**Comment by P-N-L** [comment]
I think this makes a lot of sense. The slide level might be different for different slide decks, it should be able to be considered a property of each individual document.

**Reply by jgm** [reply]
Conceptually, `slide-level` isn't metadata.  It's a processing instruction.  What I do is create a Makefile so that the instructions needed to build each document are right there with the document; I don't have to remember anything but "type `make`." Defaults files are another good solution.
